examples/next/faust-nx/wp-templates/single.tsx

```diff
@@ -13,17 +13,18 @@ const Component = (props: any) => {
 
 const query = gql`
   query GetPost($uri: ID!) {
-    post(id: $uri, idType: URI) {
+    post(id: $uri, idType: URI, isPreview: $isPreview) {
```

Comment:
I wonder if the second variable could be an optional context object with the isPreview: false instead of the isPreview directly. What do you think?